api/http/handler/endpoints/endpoint_update.go

```diff
@@ -8,7 +8,7 @@ import (
 	httperror "github.com/portainer/libhttp/error"
 	"github.com/portainer/libhttp/request"
 	"github.com/portainer/libhttp/response"
-	"github.com/portainer/portainer/api"
+	portainer "github.com/portainer/portainer/api"
```

Comment:
Look at your IDE to fix your auto-import feature, this import update is not required.